terasoluna-gfw-common-libraries/terasoluna-gfw-validator/src/main/java/org/terasoluna/gfw/common/validator/constraints/Compare.java

```diff
@@ -140,6 +140,12 @@
         EQUAL(CompareStrategy.EQ),
 
         /**
+         * Left side object must be not equal Right side object.
+         * @since 5.4.0

```

Comment:
Based on https://github.com/terasolunaorg/terasoluna-gfw/issues/711#issuecomment-314321030 discussion, it's better to change `@since 5.3.1`.